doc/old_and_new_proposal.md

```diff
@@ -662,6 +676,8 @@ following options.
     * `use_needed`  
       The created VG will match the requirements 1:1, so its size
       will be exactly the sum of all the LVs sizes.
+    * `use_vg_size`  
+      The VG will have exactly the size specified in `lvm_vg_size`.
```

Comment:
Right now, only `use_needed` is implemented. Maybe we should add `TODO` to the other two options?